### ISSUE TITLE
Level 5 packaging issues

### ISSUE THREAD

**Issue by rilwan99** [nus-cs2030/2021-s2#329]
# Summary
Hi, I'm not sure why but when I submit my code on codecrunch, the output for the test cases is empty, although I pass all the test cases when I run them on IntelliJ. Would someone be able to help me with this please? Is there something wrong with my packaging?

## Screenshots (if any): 
Insert Images here if necessary
![2021-04-25](https://user-images.githubusercontent.com/77623631/115971129-4b6d1d00-a579-11eb-95a8-7b6171c29b17.png)
![2021-04-25 (1)](https://user-images.githubusercontent.com/77623631/115971147-6049b080-a579-11eb-965a-f92b1e22b012.png)
![2021-04-25 (2)](https://user-images.githubusercontent.com/77623631/115971161-6b9cdc00-a579-11eb-8532-1db384c228a0.png)

**Comment by OuaisBien** [comment]
I think you should remove Mains from cs2030.simulator (put them in a parent directory and import all necessary classes from cs2030.simulator).

**Comment by Brandon-Chiu** [comment]
Yep, Main files should not be in the simulator directory. Also, do take note of the order of importing the classes in the Main file.

**Comment by Jason-Ng00** [comment]
You can try to remove the MainX.java from the cs2030.simulator, and import all your class in the MainX.java

**Comment by kexinnnwang** [comment]
Hi you can refer to #242 for detailed instructions
